why_postgres.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: 'A new serverless Postgres platform'
+description: "We're announcing the next major evolution for the Xata data platform, a serverless Postgres service."
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-direct-access.jpg
+  alt: Serverless Postgres Platform
+author: Monica Sarbu, Tudor Golubenco
+date: 03-18-2024
+tags: ['announcements']
+published: true
+slug: serverless-postgres-platform
+---
+
+Starting today, Xata databases and branches are also serverless Postgres databases. This means you can connect to them using any Postgres client and generally use them as you would use any other Postgres database, while **still keeping** everything that you loved about Xata: instant DB/branch creation, zero cold starts, no pausing/sleeping in case of inactivity, the full-text search engine, the file attachments, and more. And yes, the Xata free tier also applies to the Postgres databases, making it one of the most generous free tiers for Postgres in the industry.
```

Comment:
```suggestion
Starting today, Xata databases and branches are also serverless Postgres databases. This means you can connect to them using any Postgres client and generally use them as you would use any other Postgres database, while **still keeping** everything that you love about Xata: instant DB/branch creation, zero cold starts, no pausing/sleeping in case of inactivity, the full-text search engine, the file attachments, and more. And yes, the Xata free tier also applies to the Postgres databases, making it one of the most generous free tiers for Postgres in the industry.
```

Reply:
Changing as its implies they no longer love us but of course they do ;-)